tests/vm

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -eux
+
+# Install LXD
+install_lxd
+
+# Configure LXD
+lxd init --auto
+
+IMAGE="${TEST_IMG:-ubuntu-minimal-daily:24.04}"
+
+# Launch test instance
+lxc launch "${IMAGE}" vm1 --vm
+waitInstanceReady vm1
```

Comment:
```suggestion
waitInstanceBooted vm1
```

Reply:
> taking ~2h feels like a bug as I'd expect lxc stop to timeout before that long, no?

I think the timeout is indeed very long so this should be expected

> That said, I think the context deadline bit might be addressed by waiting for complete boot before doing the shutdown.

Yeah, that makes sense, thanks!